java/ql/src/change-notes/2025-06-10-reduce-precision-for-building-cmdline-with-string-concatenation.md

```diff
@@ -0,0 +1,3 @@
+category: queryMetadata
```

Comment:
```suggestion
---
category: queryMetadata
```
We need this at the top of the file so that the script that combines change notes can work properly.